.github/workflows/vectorlint.yml

```diff
@@ -0,0 +1,27 @@
+name: Vector Lint
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron:  '15 */12 * * *'
+
+jobs:
+  build:
+    name: Lint vectors/schemas
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v5
```

Comment:
```
    with:
      go-version-file: go.mod
```

The default is "whatever we have around in a cache, I guess"

Reply:
Ahh! Noted. I re-arranged to use this approach.